pkg/db/db.go

```diff
@@ -2,7 +2,6 @@ package db
 
 import (
 	"fmt"
-	"log"
```

Comment:
Honestly , that sounds like too much work. what are you thoughts on replacing the existing error messages with a custom error like this
```go
package main

import (
	"fmt"
	"path"
	"runtime"
)

func newError(message string) error {
	_, file, line, _ := runtime.Caller(1)
	fileName := path.Base(file)
	return fmt.Errorf("%s:%d: %s", fileName, line, message)
}

func main() {
	err := newError("something went wrong")
	fmt.Println(err)
}
 return fmt.Errorf("%s:%d: %s", fileName, line, message)
}
```
So when this gets called we'll have an output that displays the same information 
![image](https://github.com/theredditbandit/pman/assets/85390033/228997b4-093e-4bdf-97a4-d4d011bf9c86)

Reply:
The debug mode would be overkill.

When you have an error, you want it to have it reported.

I don't think that having the line reported has a need, a go program is made to be compiled.

What you can do is to wrap error with `fmt.Errorf` everywhere you think it would bring extra information.

Reply:
hmm, I agree a separate debug mode would be overkill.

But I still feel that having a custom error type that returns line number and file name would make things easier when debugging reported errors or errors I might encounter while developing. 

I think I will add it myself down the line.